src/index.js

```diff
@@ -30,22 +17,73 @@ function renderHtml() {
     .sort((a, b) => a.index - b.index)
     .forEach((task) => {
       listParent.innerHTML += `
-      <li>
+      <li draggable=true>
         <div class="content">
           <input class="check" type="checkbox" ${
-            task.completed ? 'checked' : ''
-          }/>
+  task.completed ? 'checked' : ''
+}/>
           <input class="input" type="text" value='${
-            task.description
-          }' readonly />
+  task.description
+}' readonly />
         </div>
         <div class="actions">
           <span class="material-icons drag">more_vert</span>
-          <span class="material-icons hide">delete_outline</span>
+          <span class="material-icons hide" id="delete">delete_outline</span>
         </div>
       </li>
       `;
     });
 }
 
 renderHtml();
+
+// adding and removing
+
+returnIcon.addEventListener('click', () => {
+  if (textInput.value.length === 0) {
+    alert('Please Enter a To Do');
+  } else {
+    tasks.push({
+      description: textInput.value,
+      completed: false,
+      index: tasks.length,
+    });
+
+    localStorage.setItem('todos', JSON.stringify(tasks));
+
+    const updateTasks = () => {
+      listParent.innerHTML = '';
+      tasks.forEach((task) => {
+        listParent.innerHTML += `
+     <li draggable ="true" id="${task.index}">
+       <div class="content">
+         <input class="check" type="checkbox" ${
+  task.completed ? 'checked' : ''
+}/>
+         <input class="input" type="text" value='${
+  task.description
+}' readonly />
+       </div>
+       <div class="actions">
+         <span class="material-icons drag">more_vert</span>
+         <span class="material-icons" id="deleteicon">delete_outline</span>
+       </div>
+     </li>
+     `;
+      });
+      const deletButton = document.querySelectorAll('#deleteicon');
+
+      deletButton.forEach((x) => {
+        x.addEventListener('click', () => {
+          const id = Number(x.parentNode.parentNode.id);
+          tasks = tasks.filter((task) => task.index !== id);
+          updateTasks();
+          localStorage.setItem('todos', JSON.stringify(tasks));
```

Comment:
- [ ] When I delete one task, it does not currently update the indexes to match the new list size, kindly do this as it's one of the requirements

![image](https://user-images.githubusercontent.com/73140461/149192919-aa5bb856-92f6-4431-aac7-d021f1d37cd8.png)